Xamarin.Forms.Platform.iOS/Renderers/ImageRenderer.cs

```diff
@@ -202,9 +202,7 @@ public sealed class FontImageSourceHandler : IImageSourceHandler
 			{
 				var iconcolor = fontsource.Color != Color.Default ? fontsource.Color : Color.White;
 				var imagesize = new SizeF((float)fontsource.Size, (float)fontsource.Size);
-				var hasFontFamily = fontsource.FontFamily != null && UIFont.FamilyNames.Contains(fontsource.FontFamily);
-				var font = hasFontFamily ?
-					UIFont.FromName(fontsource.FontFamily, (float)fontsource.Size) :
+				var font = UIFont.FromName(fontsource.FontFamily ?? string.Empty, (float)fontsource.Size) ??
```

Comment:
As long as this does not throw for an invalid name, we should be good.

Reply:
Confirmed: `UIFont.FromName("carrots", 12) == null`

Reply:
@mattleibow passing in a null for the font name throws a NullReferenceException hence the null guard where we pass in string.Empty if the FontFamily is null... if the Font is not located it returns a default (null) UIFont which is guarded to provide the default system font.

Reply:
@dansiegel ✅ Merge!